lib/network.sh

```diff
@@ -38,12 +38,12 @@ export POD_CIDR=${POD_CIDR:-"192.168.0.0/18"}
 PROVISIONING_IPV6=${PROVISIONING_IPV6:-false}
 IPV6_ADDR_PREFIX=${IPV6_ADDR_PREFIX:-"fd2e:6f44:5dd8:b856"}
 
+export LIBVIRT_FIRMWARE=${LIBVIRT_FIRMWARE:uefi}
```

Comment:
there is a typo here, you're missing a "-" to default it should be ":-"

Reply:
Thanks!